integration_with_openshift_container_platform/_topics/ocp-metrics.md

```diff
@@ -52,15 +52,15 @@ To create these accounts:
     **metrics-deployer** account to have **edit** permissions in the
     **openshift-infra** project:
 
-        $ oadm policy add-role-to-user \
```

Comment:
These changes must have been brough in in a merge?  They should not be in the PR.  Perhaps squash your commits?

Reply:
I have squahed my commits.